tests/staticFilesServer.test.js

```diff
@@ -0,0 +1,158 @@
+/* eslint-disable indent */
+/* eslint-disable max-len */
+'use strict';
+
+const axios = require('axios');
+const path = require('path');
+const { faker } = require('@faker-js/faker');
+const fs = require('fs');
+const { Server } = require('http');
+const { createServer } = require('../src/createServer.js');
+
+const PORT = 5701;
+const HOST = `http://localhost:${PORT}`;
+
+function generateRandomCSS() {
+  return `
+    /* Random CSS Generated by Faker */
+
+    .${faker.lorem.word()} {
+      color: ${faker.internet.color()};
+      font-size: ${faker.number.int({
+    min: 12, max: 24,
+  })}px;
+    }
+
+    .${faker.lorem.word()} {
+      background-color: ${faker.internet.color()};
+      border: 2px solid ${faker.internet.color()};
+      padding: ${faker.number.int({
+    min: 5, max: 20,
+  })}px;
```

Comment:
Do we really need to generate all this data randomly?
to prevent cheating it is enough to have just 1 random word in a file

Reply:
why not?)